test/fixtures/split-translations/src/root.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import React from 'react';
+import Router, {Route, Switch, Link} from 'fusion-plugin-react-router';
+import {Translate} from 'fusion-plugin-i18n-react';
+import {split} from 'fusion-react-async';
```

Comment:
nit: seems like we don't use this import in this file. I'm surprised linters are passing, maybe our config is incorrect?

Reply:
Hmm, maybe because this is a fixture